akka-docs/src/main/paradox/assets/js/warnOldDocs.js

```diff
@@ -0,0 +1,186 @@
+jQuery(document).ready(function ($) {
+
+  function initOldVersionWarnings($) {
+    $.get("//akka.io/versions.json", function (akkaVersionsData) {
```

Comment:
There's no error reporting when this fails or is disallowed by the browser - that seems OK though, as we gracefully handle it by not showing a banner